src/components/MainContent/Components.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Element } from 'react-scroll/modules';
+
+export interface ContentProps {
+  children:
+    | void
+    | string
+    | JSX.Element
+    | JSX.Element[]
+    | (JSX.Element | undefined)[]
+    | (JSX.Element | undefined);
+}
+
+export interface SectionProps extends ContentProps {
+  name: string;
+}
+
+export const Section = ({ name, children }: SectionProps) => (
+  <Element name={name}>
+    {children}
+    <style jsx>{`
+      padding-left: 0.5em;
+    `}</style>
+  </Element>
+);
+
+export const MainHeader = ({ children }: ContentProps) => (
+  <header>
+    <h1>{children}</h1>
+    <style jsx>{``}</style>
+  </header>
+);
+export const SubHeader = ({ children }: ContentProps) => (
+  <header>
+    <h2>{children}</h2>
+    <style jsx>{``}</style>
+  </header>
+);
+
+export const NonBulletList = ({ children }: ContentProps) => (
+  <ul>
+    {children}
+    <style jsx>{`
+      * {
```

Comment:
Is a `*` selector the best one to use here? I don't like that it takes priority over everything else; we might need to think about how children are represented in this component. For example, we could pass an array of strings into this component, map over that to create children, and then style the resulting `li` components.